descriptor-template.json

```diff
@@ -0,0 +1,33 @@
+{
+    "package": {
+        "name": "pega-helm-charts",
+        "repo": "pega-helm-charts",
+        "subject": "kishor",
+        "vcs_url": "https://github.com/scrumteamwhitewalkers/pega-helm-charts-1",
```

Comment:
Should this be here?

Reply:
Fixed. Changed to Pegasystems repository values.